lib/services/configuration.go

```diff
@@ -46,6 +41,18 @@ type ClusterConfiguration interface {
 	GetClusterConfig(opts ...MarshalOption) (ClusterConfig, error)
 	// SetClusterConfig sets services.ClusterConfig on the backend.
 	SetClusterConfig(ClusterConfig) error
+}
+
+// LocalClusterConfiguration manages cluster configuration on auth server
+type LocalClusterConfiguration interface {
```

Comment:
can this be in `lib/auth`?

Reply:
Even though this is a server-side view, it will cause import cycle if moved to lib/auth. I will rename it to `ServerClusterConfiguration` in acc. with your suggestion.